Add testing file to Travis

jacklecroy opened the pull request:
Tests can be put into tests.py and we can merge this once there are enough of them.

Fixes #11

jacklecroy committed:
Add tests file to Travis

Tests can be put into tests.py

RyanSchw commented:
You're going to need to install the python requirements (#10) as part of the `travis.yaml` file

jacklecroy committed:
Merge remote-tracking branch 'origin/master' into feature_add_testing

jacklecroy committed:
Adds mypy for static type checking

Right now there are fatal syntax errors
so it will not display any warnings or
errors regarding type.

RyanSchw commented:
Closed as insufficient - mypy can be added later when we use `unittest` to properly test our python files.
